### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by adrfantini** [ericman314/UnitMath#57]
Hello, maybe this is a very dumb question, but I can't figure out how to test this package.

If I manually load it in Node using `require`, after installing with `npm i unitmath`:
```
> require('unitmath')
/dev/shm/unitmath/node_modules/unitmath/index.js:1
import UnitMath from './src/Unit.js'
^^^^^^

Uncaught:
SyntaxError: Cannot use import statement inside the Node.js REPL, alternatively use dynamic import:
>
```

Node using `import`:
```
> await import ("unitmath")
(node:145890) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
Uncaught [SyntaxError: Unexpected token (1:13)
] {
  pos: 13,
  loc: Position { line: 1, column: 13 },
  raisedAt: 23
}
```

Deno using `import`:
```
> await import('unitmath')
Uncaught TypeError: 'import', and 'export' cannot be used outside of module code at file:///dev/shm/unitmath/node_modules/.deno/unitmath@1.0.1/node_modules/unitmath/index.js:1:1

  import UnitMath from './src/Unit.js'
  ~~~~~~
    at async <anonymous>:1:22
```

The same seem to occur when running a `.js` or `.mjs` file with the following content:
```js
const unit = require('unitmath')

let a = unit('5 m').div('2 s')   // 2.5 m / s
let b = unit('40 km').to('mile')  // 24.8548476894934 mile
b.toString({ precision: 4 })          // "24.85 mile"
```

What am I doing wrong?

**Comment by adrfantini** [comment]
This is fixed by adding
```json
  "exports": {
    ".": {
      "import": "./dist/UnitMath.js",
      "require": "./dist/UnitMath.js"
    }
  },
```
to the `package.json`.

I can open a PR if needed

**Comment by ericman314** [comment]
Thank you for reporting, and especially for your solution! I did not know about the `exports` keyword.

This should be fixed now in v1.0.2.

**Comment by adrfantini** [comment]
> Thank you for reporting, and especially for your solution! I did not know about the `exports` keyword.
> 
> This should be fixed now in v1.0.2.

No problem, thank you!
